tasks/sections/setup_systemd_config.yml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Seems that it's not about `systemd config`.

As I see, this task does this things:

* manages instance parameters in runtime
* checks if instance restart is needed
* places instance and application configuration

What do you think about splitting this stage into 
* `check_restart_is_needed` - in the future this stage can set up variable that indicates that instance restart is needed. It can be used to replace `Restart instance systemd service` task. BTW, it seems that it can't be handler since we allow to replace scenario parts, what if somebody uses custom commands to start instances? 
* `update_configurations`

Reply:
This is one logical block, so it is better not to divide it. However, I renamed to a more correct name